source/module_base/global_function.cpp

```diff
@@ -112,7 +115,8 @@ void DONE(std::ofstream &ofs,const std::string &description, const bool only_ran
 void TEST_LEVEL(const std::string &name)
 {
     bool disable = true;
-    if (disable) return;
+    if (disable)
+        return;
```

Comment:
This format change seems unnecessary, only two formats is allowed for "if" sentence
"if(disable) return;"
or
"
if(disable)
{
    return;
}
"

Reply:
@dyzheng I think the format in proposed changing is accectable.